Controller/Adminhtml/YotpoController/YotpoController.php

```diff
@@ -50,13 +50,14 @@ public function execute()
         return;
       }
       $offset = 0;
-      $orderStatuses = $this->_config->getCostumeOrderStatus();
+      $orderStatuses = $this->_config->getCustomOrderStatus();
       if ($orderStatuses == null) {
           $orderStatuses = array(\Magento\Sales\Model\Order::STATE_COMPLETE);
       } else {
-        $orderStatuses = array_map('strtolower', explode(' ', $orderStatuses));
+        $orderStatuses = array_map('strtolower', explode(',', $orderStatuses));
```

Comment:
In Magento when multiselect value is saved from configuration, the values in database are stored as quoma seperated instead of space separated.